src/commands/statusline.ts

```
@@ -437,6 +445,21 @@ export const statuslineCommand = define({
 		if (Result.isSuccess(mainProcessingResult)) {
 			const statusLine = mainProcessingResult.value;
 			log(statusLine);
+
+			// Save Claude Code cost data if available and enabled
+			if (ctx.values.saveCost && hookData.cost?.total_cost_usd != null) {
```

ESLint

[error] Unexpected any value in conditional. An explicit comparison or type conversion is required.

(ts/strict-boolean-expressions)